lib/matplotlib/patheffects.py

```diff
@@ -386,6 +386,8 @@ def draw_path(self, renderer, gc, tpath, affine, rgbFace):
         affine = self._offset_transform(renderer, affine)
         self.patch._path = tpath
         self.patch.set_transform(affine)
-        self.patch.set_clip_box(gc._cliprect)
-        self.patch.set_clip_path(gc._clippath)
+        self.patch.set_clip_box(gc.get_clip_rectangle())
+        clip_path = gc.get_clip_path()
```

Comment:
This seems to be more correct and possibly fixes bugs?  Is it worth tracking down what those might be?